test/test_config_converter.py

```diff
@@ -27,6 +27,14 @@ def check_equality(config_name):
     assert expected == observed
 
 
+def test_types():
+    for prg in {
+            'config_script.py',
+            'config_converter/config_mapper/config_mapper.py'
+    }:
+        subprocess.run(['python3', '-B', '-m', 'pytype', prg], check=True)
```

Comment:
PyType tests! This is great that we can enforce the types for the functions with this test!